templates/projects/tasks_browse.html

```diff
@@ -187,6 +190,9 @@
                 {% if 'completed_by' in filter_data.display_columns %}
                 <th class="{{'sort-asc' if filter_data.order_by['completed_by'] == 'asc'}} {{'sort-desc' if filter_data.order_by['completed_by'] == 'desc'}} sortable" data-sort="completed_by">Completed By</th>
                 {% endif %}
+                {% if 'assigned_users' in filter_data.display_columns %}
+                <th class="{{'sort-asc' if filter_data.order_by['assigned_users'] == 'asc'}} {{'sort-desc' if filter_data.order_by['assigned_users'] == 'desc'}} sortable" data-sort="assigned_users">Assigned Users</th>
```

Comment:
This change should actually go in tasks_browse.webpack.ejs as shown [here](https://github.com/bloomberg/pybossa-default-theme/blob/main/templates/projects/tasks_browse.webpack.ejs#L190).

The file `tasks_browse.html` is auto-generated by the ejs file during compilation. Also, to disable the sort, the code can use::

```
<th>Assigned Users</th>
```

Reply:
Thanks, will update .ejs.
I could manage to get basic sorting working for assigned users. Hence, enabled sorting.